CONTRIBUTING.md

```diff
@@ -5,39 +5,42 @@ Please familiarize yourself with the SPDX License List and its supporting docume
 * [SPDX Specification](https://spdx.org/specifications): It is helpful to be familiar with certain sections of the SPDX Specification thatuse or deal with the SPDX License List. In particular: sub-sections related to license information in Section 3, 4, and 6; Appendices II, IV, and V.  
 
 # How to contribute/participate: general
-The SPDX License List is maintained by the SPDX Legal Team. Work and discussion is primarily done via the mailing list and bi-weekly calls. In particular, the bi-weekly calls are used to discuss topics and issues that may be difficult to only discuss via email.
-You can join the mailing list and otherwise manage your subscription [via spdx-legal mailman](https://lists.spdx.org/mailman/listinfo/spdx-legal). Information on the bi-weekly calls and the working area for the SPDX Legal Team, can be found on the [SPDX legal team wiki](https://wiki.spdx.org/view/Legal_Team)
+The SPDX License List is maintained by the SPDX Legal Team. Work and discussion is primarily done via:
+* mailing list: our traditional form of communication. Meeting reminders, minutes, updates are posted there, as well as discussion.
+* this Github repo: comments and issues and PRs related to specific changes to the files that comprise the SPDX License List, e.g., update a URL, recommend additional markup for matching purposes. In particular, new license requests are tracked here.
+* bi-weekly calls: used to discuss topics and issues that may be difficult to only discuss via email or via Github.
+You can join the mailing list and otherwise manage your subscription [via spdx-legal mailman](https://lists.spdx.org/mailman/listinfo/spdx-legal). 
```

Comment:
I think it would make more sense to make this line part of the "mailing list" bullet point.

Reply:
good idea, just changed